e2e/conf/local.conf.js

```diff
@@ -13,7 +15,7 @@ nightwatch_config = {
 
   test_settings: {
     default: {
-      globals_path: "globals.js",
+      globals_path: "test-globals.js",
```

Comment:
Can this be reverted back to globals.js. You can add the -env switch to choose dev or test. (see package.json for the specific commands.)